scripts/flow/react-relay-hooks.js

```diff
@@ -39,7 +39,7 @@ declare module 'ReactFlightDOMRelayServerIntegration' {
   import type {JSResourceReference} from 'JSResourceReference';
 
   declare export opaque type Destination;
-  declare export opaque type BundlerConfig;
```

Comment:
We can't change these without changing the internal code in Meta's codebase. So someone at Meta needs to do that both at the same time.

Reply:
Not sure if its connected. But Relay has cleaned up its flight implementation: https://github.com/facebook/relay/commit/689dcb170a7500f99cf9ee04e5be91ee0c4925cb

Reply:
Reverted to `BundlerConfig` and added a TODO.